trustfall_core/src/interpreter/helpers.rs

```diff
@@ -63,6 +66,110 @@ pub fn resolve_coercion_with<'vertex, Vertex: Debug + Clone + 'vertex>(
 }
 
 /// Helper for making property resolver functions based on fields.
+///
+/// Generally used with [`resolve_property_with`].
+///
+/// Retrieves a [FieldValue] from a vertex by converting it to the proper type,
```

Comment:
I'm not a docs expert, do we generally want to put type names in code blocks or is there some reason it's better not to here?
```suggestion
/// Retrieves a [`FieldValue`] from a vertex by converting it to the proper type,
```

Reply:
You're right, I though it automatically code-formatted when it was a link to code.